lib/shelley/src/Cardano/Wallet/Shelley/Pools.hs

```diff
@@ -1,10 +1,8 @@
-{-# OPTIONS_GHC -Wno-unrecognised-pragmas #-}
-
+{-# LANGUAGE BlockArguments #-}
```

Comment:
(I'm still not foo fond of it, as it messes up the "whitespace separates function arguments" property. 😅)